modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
Yes, and in that case it would be up to `import`ing code to rename that to avoid the conflict, right?
```js
import {Node as 'OSMNode'} from '/js/lib/id/node'
```
(still trying to wrap my head around ES6 modules)